### DISCUSSION TITLE
What does Siglo do?

### DISCUSSION THREAD

**Discussion by FenellaKW** [theironrobin/siglo#88] [Q&A]
Hi All,
I know it sounds like a bit of a stupid question but I can't seem to find any answer to list exactly what is supported by Siglo currently.
Perhaps there's some documentation I've missed? If so can you point me in the right direction?

I know I can currently connect my PineTime to Siglo to update the time on the watch and also push new firmware.
But what else (if anything) does it do?

- Can I currently control music playing on my device?
- Can I receive notifications?
- Can I access any location information?
- Anything else?

Thanks.

**Comment by kq98** [comment]
1. no but PR is pending
2. currently broken
3. no
4. not to my knowledge

**Comment by theironrobin** [comment]
@kq98 this is really late (sorry i moved cities and then life got in the way) I think i want siglo to be primarily a CLI Project, and then the gui front-end is really just reporting the background processes that are CLI calls. I will take a look at your PR soon, after taking care of some housekeeping for this project
